### ISSUE TITLE
[Community] Member Profile: Ian Whitney

### ISSUE THREAD

**Issue by hargunkaur286** [layer5io/layer5#6047]
**Current Behavior**

@ianrwhitney has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @ianrwhitney  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @ianrwhitney
- Twitter: 
- LinkedIn: https://www.linkedin.com/in/ian-whit/
- Layer5 Cloud:  https://meshery.layer5.io/user/3e31adc3-6062-41f5-83f3-21aa2cbedd24 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

**Comment by hargunkaur286** [comment]
@ianrwhitney Do you mind adding rest of your details in here?

**Comment by samnmbuguah** [comment]
@hargunkaur286 I would like to work on this issue, could you assign it to me?

**Comment by sudhanshutech** [comment]
@samnmbuguah are you still up to do this then lets assign it to you.

**Comment by samnmbuguah** [comment]
> @ianrwhitney would you mind your sharing your
> 
> * Twitter
> * Profile Picture
> * Bio

 @ianrwhitney the above details and your location are required to complete your member profile. Could you please share them?

**Comment by leecalcote** [comment]
Did we get this issue completed?

**Comment by samnmbuguah** [comment]
> Did we get this issue completed?

No, I did not get the required information to complete the profile.
